src/others/contexts/file-downloader.tsx

```diff
@@ -97,8 +97,16 @@ export const FileDownloaderContextProvider: FunctionComponent = ({ children }) =
 
       const csvRow = [requestDate];
 
+      const categoryToAmount: { [id: string]: number } = {};
       cityRequests.forEach((request) => {
-        csvRow.push(request.amount + "");
+        categoryToAmount[request.category] = request.amount;
+      });
+      uniqueCategories.forEach((category) => {
+        if (category in categoryToAmount) {
```

Comment:
**issue**

I think that checking for truthiness of `(categoryToAmount[category])` would give us more value:

"If you set a property to [undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined) but do not delete it, the in operator returns true for that property."
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

Reply:
Interesting point. 

Although I think the current version is better, since if categoryToAmount[category] === undefined and you go to "else" statement, you are hiding a real problem in the code.
While the version right now will display exactly the data in the dict and the potential problem will be visible and hopefully fixed.
So will keep for now the current version.